bin/backup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Takes a backup of the error reporter DB in the current dir
+if [[ -z "$1" ]]; then
+    >&2 echo "The name of the file to backup is required"
+    exit 1
+fi
+
+
+SCRIPTPATH=$(cd "$(dirname "$0")"; pwd -P)
+SOURCE_DIR=$(cd "$SCRIPTPATH" && cd .. && pwd -P)
+
+source ${SOURCE_DIR}/.env
+
+PROJECT_NAME=reports
+PG_DOCKER_NAME=$PROJECT_NAME"_postgres_1"
+
+docker exec -t $PG_DOCKER_NAME pg_dumpall -c -U ${DB_USER} | gzip > $1
```

Comment:
We need to stop the instance to have a consistent backup
We had a corrupt Django table (thankfully an ORM cache) from a previous migration where we took a backup of a live instance. I wonder if there's a way to do this from Django instead so we can backup live?

Reply:
It looks to add a set of commands to `manage.py` so we can just run those...

Reply:
I'll see if I can get this going.